cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,118 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('display multiple emails "unverified" and "verified"', () => {
```

Comment:
Is the `bold` font on the emails meant for the primary email? just making sure

Reply:
That's right. In the sidebar the primary email is set in bold to make it stand out a little mroe than the other email addresses.